lib/views/main.hbs

```diff
@@ -3,6 +3,7 @@
     <head>
         <title>Gemini</title>
         <link rel="stylesheet" type="text/css" href="main.css">
+        <meta name="isRunImmediately" content="{{isRunImmediately}}">
```

Comment:
looks like hack, very dirty hack :) I understand that you can write to meta any information but this tag was created for side description, not for our hacks.